src/Lecture1.hs

```diff
@@ -77,7 +80,17 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z =
+  let max'
+        | x > y && x > z = x
+        | y > z = y
+        | otherwise = z
+      min'
+        | x < y && x < z = x
+        | y < z = y
+        | otherwise = z
```

Comment:
You can also use standard functions `max` and `min` to define variables a bit shorter ✂️